Optimize (CQA) - Mng content - Recovering (#4874)
theforeman/foreman-documentation: guides/common/modules/con_recovering-corrupted-content.adoc

= Recovering corrupted content

[role="_abstract"]

> Resynchronize corrupted repositories on {ProjectServer}, verify checksums on {SmartProxies}, and republish repository or content view metadata when distributions no longer match stored content.

Reviewer: ```suggestion
Resynchronize corrupted repositories on {ProjectServer}, verify checksums on {SmartProxyServers}, and republish repository or content view metadata when the meta data of repositories no longer matches stored content.
```

Author: ```suggestion
Resynchronize corrupted repositories on {ProjectServer}, verify checksums on {SmartProxyServers}, and republish repository or content view metadata when the repository metadata no longer matches stored content.
```
